src/systems/model_photo_shoot/ModelPhotoShoot.hh

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2018 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_GAZEBO_SYSTEMS_MODELPHOTOSHOOT_HH_
+#define IGNITION_GAZEBO_SYSTEMS_MODELPHOTOSHOOT_HH_
+
+#include <sdf/sdf.hh>
+
+#include <memory>
+
+#include <ignition/gazebo/Model.hh>
+#include <ignition/gazebo/System.hh>
+#include <ignition/gazebo/World.hh>
+#include <ignition/rendering/Camera.hh>
+#include <ignition/rendering/RenderingIface.hh>
+#include <ignition/transport/Node.hh>
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace systems
+{
+  // Forward declarations.
+  class ModelPhotoShootPrivate;
+
+  /// \brief System that takes snapshots of an sdf model
```

Comment:
Can you document what this system does and its parameters? See https://github.com/ignitionrobotics/ign-gazebo/blob/ee5e37699eaed8728431bae3b66a73561fe34ff0/src/systems/thruster/Thruster.hh#L35-L91 as an example.

Reply:
Added.

Reply:
Looks great! Thanks!